packages/@okta/vuepress-site/docs/guides/enable-cors/overview/index.md

```diff
@@ -2,133 +2,22 @@
 title: Overview
 ---
 
-[Cross-Origin Resource Sharing (CORS)](http://en.wikipedia.org/wiki/Cross-Origin_Resource_Sharing) is a mechanism that allows a web page to make an AJAX call using [XMLHttpRequest (XHR)](http://en.wikipedia.org/wiki/XMLHttpRequest) to a domain that is  different from the one from where the script was loaded.  Such "cross-domain" requests would otherwise be forbidden by web browsers, per the [same origin security policy](http://en.wikipedia.org/wiki/Same_origin_policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether or not to allow the cross-origin request
+[Cross-Origin Resource Sharing (CORS)](https://www.w3.org/TR/cors/) is a mechanism that allows a web page to make an AJAX call using [XMLHttpRequest (XHR)](https://xhr.spec.whatwg.org/) to a domain that is different than the domain where the script was loaded. Such cross-domain requests would otherwise be forbidden by web browsers, per the [same origin security policy](https://developer.mozilla.org/en-US/docs/Web/Security/Same-origin_policy). CORS defines a [standardized](http://www.w3.org/TR/cors/) way in which the browser and the server can interact to determine whether or not to allow the cross-origin request.
```

Comment:
"per" - usage when 'according to something' should be 'as per'. I might suggest casting that aside as it's fairly formal language. Something like "would otherwise be forbidden by web browsers as indicated by the same origin security policy"?

Reply:
sure - I didn't do a lot of reading of the existing text - just replaced links to wikipedia with links to the specs. I'll take care of that. thanks.